examples/NAS-Envelopenet-trainingcontainer/model_constructor.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
You probably don't need __future__ imports since you are using python3 anyways. Same goes for every import like this.